go/test/endtoend/onlineddl/vtgate_util.go

```diff
@@ -92,6 +92,17 @@ func CheckRetryMigration(t *testing.T, vtParams *mysql.ConnParams, shards []clus
 	}
 }
 
+// CheckRetryPartialMigration attempts to retry a migration where a subset of shards failed
```

Comment:
The other one seemed to be expecting different output (and pretty sure it was failing, otherwise I would not have added another function as I'm trying to limit additions):
```
	if expectRetryPossible {
		assert.Equal(t, len(shards), int(r.RowsAffected))
	} else {
		assert.Equal(t, int(0), int(r.RowsAffected))
	}
```

Reply:
Confirmed that the CheckRetryMigration fails in the partial case (in this case there's 2 shards, one succeeded):
```
        	Error Trace:	vtgate_util.go:89
        	            				onlineddl_vrepl_test.go:581
        	Error:      	Not equal:
        	            	expected: 2
        	            	actual  : 1
        	Test:       	TestSchemaChange/PlannedReparentShard_via_throttling_1/2/Check_tablet_post_PRS
```